completions/_sops

```diff
@@ -0,0 +1,24 @@
+# 3rd party completion loader for commands using           -*- shell-script -*-
+# version 1 of the https://cli.urfave.org library.
+#
+# This serves as a fallback in case the completion is not installed otherwise.
+
+# https://github.com/urfave/cli/blob/v1-maint/docs/v1/manual.md#bash-completion
+# https://github.com/urfave/cli/blob/v1-maint/autocomplete/bash_autocomplete
+_comp_cmd__urfave_cli_v1()
```

Comment:
Not sure if this is according to the naming guideline, but at least it is "in spirit".

Reply:
I think this is OK.